.ci.yaml

```diff
@@ -1377,14 +1377,26 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: analyzer_benchmark
 
-  - name: Linux_android android_defines_test
+  - name: Linux android_defines_test
```

Comment:
I believe Linux_android is required for the dashboard to select the proper icon.

Reply:
Hmm. Okay let me verify that will work here.